components/ArtPiecePreview/ArtPiecePreview.js

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function ArtPiecePreview({ artPiece }) {
+  console.log(artPiece);
```

Comment:
You always should remove console.log's before pushing it to the github-world :)

Reply:
Removed all console logs ✅